src/commerce-handler.js

```diff
@@ -9,66 +10,140 @@ CommerceHandler.prototype.logCommerceEvent = function(event) {
         event
     );
     expandedEcommerceEvents.forEach(function(expandedEvent) {
-        var optimizelyEvent = {
-            type: 'event',
-            eventName: event.EventName,
-            tags: {}
-        };
-        optimizelyEvent.tags = expandedEvent.EventAttributes || {};
-        if (
-            event.EventCategory ===
-                mParticle.CommerceEventType.ProductPurchase ||
-            event.EventCategory === mParticle.CommerceEventType.ProductRefund
-        ) {
-            if (expandedEvent.EventName.indexOf('Total') > -1) {
+
+        if (optimizelyEvents.events[expandedEvent.EventName]) {
```

Comment:
similar to on initialization and the events handler, do a check for `window.optimizely` here instead